x/uibc/quota/keeper/grpc_query.go

```diff
@@ -28,23 +28,23 @@ func (q Querier) Params(goCtx context.Context, _ *uibc.QueryParams) (
 	return &uibc.QueryParamsResponse{Params: params}, nil
 }
 
-// Quota returns quotas of denoms.
-func (q Querier) Quota(goCtx context.Context, req *uibc.QueryQuota) (
-	*uibc.QueryQuotaResponse, error,
+// Outflows returns quotas of denoms.
```

Comment:
```suggestion
// Outflows returns quota outflow of denoms.
```

Reply:
should be a bit different. I will update